### ISSUE TITLE
Intel quicksync transcoding broken

### ISSUE THREAD

**Issue by MattSpec** [jellyfin/jellyfin#2795]
**FFmpeg Logs**
[hwupload @ 0x55a825959600] A hardware device reference is required to upload frames to.
[Parsed_hwupload_1 @ 0x55a82599eac0] Query format failed for 'Parsed_hwupload_1': Invalid argument
Error reinitializing filters!
Failed to inject frame into filter network: Invalid argument
Error while processing the decoded data for stream #0:0
[aac @ 0x55a82578f5c0] Qavg: 59444.480
[aac @ 0x55a82578f5c0] 2 frames left in the queue on closing
Conversion failed!

**Server System (please complete the following information):**
 - OS: ubuntu 19.10
 - Jellyfin Version: 10.5.3
 - Hardware settings & device: intel quicksync - intel n4100 UHD Graphics 600
 - Reverse proxy: apache

**Comment by nyanmisaka** [comment]
Will be fixed in  https://github.com/jellyfin/jellyfin/pull/2785

**Comment by MattSpec** [comment]
Ill try vaapi a bit later today. This error has only surfaced after upgrading from 10.5.2 to 10.5.3. Previously I had selected qsv and hardware encoding and it seemed to be working.

**Comment by nyanmisaka** [comment]
> Ill try vaapi a bit later today. This error has only surfaced after upgrading from 10.5.2 to 10.5.3. Previously I had selected qsv and hardware encoding and it seemed to be working.

Yes, there are some flaws in this version and will be fixed soon in the next version. However, QSV does not yet support Linux, even if you choose hardware acceleration, it will fall back to software coding.
